src/aboutkuzya.cpp

```diff
@@ -0,0 +1,56 @@
+#include "aboutkuzya.h"
+#include "ui_aboutkuzya.h"
+
+#include <QDebug>
+#include <QDir>
+
+AboutKuzya::AboutKuzya(QVersionNumber verKuzia, QDate buildDate, QString fileName, QWidget *parent) :
+  mVerKuzia(&verKuzia), mBuildDate(&buildDate), mFileName(&fileName), QDialog(parent),
+  ui(new Ui::AboutKuzya)
+{
+  ui->setupUi(this);
+
+  ui->textInfLabel = new QLabel(this);
+
+  checkFile();
+  readAuthors();
+  setVerAndDate();
+}
+
+//checks file
+void AboutKuzya::checkFile()
+{
+  mFile = new QFile (*mFileName);
+
+  if(!QFile::exists(*mFileName))
+  {
+      return;
+  }
+  if (!mFile->open(QIODevice::ReadOnly | QIODevice::Text))
+  {
+      return;
+    }
+}
+
+//reading AUTHORS file
+void AboutKuzya::readAuthors()
+{
+  mIn = new QTextStream(mFile);
```

Comment:
Read authors will be called regardless of checkFile() call state. I suggest to merge this two nethods to one. In that new method you could check for authors file and skip reading if it is not exist.